ui/src/app/utils/runtime-utils.tsx

```diff
@@ -171,6 +171,37 @@ export const findMostSevereDiffState = (
   );
 };
 
+export const applyUpdate: UpdateMessaging = {
```

Comment:
no changes here - just split out for ease of understanding and testing

Reply:
Thanks!